### ISSUE TITLE
Crash on startup for the fabric 1.18.2 release

### ISSUE THREAD

**Issue by Spirit72** [GiantLuigi4/Smaller-Units#195]
Hello! I'm trying to use the fabric release of Smaller-Units 1.18.2 (and i put cardinal components api in there), but minecraft keeps crashing. and i'm not sure why since it just says "exit code 1"

(i only have fabric-api, Smaller-Units and Cardinal-Components-API downloaded)

am i by chance missing a mod? thanks :p

**Comment by GiantLuigi4** [comment]
For some reason, either loom or mixin failed to function properly while I was building the mod, so it just can’t load at all from what I understand…
Also for some reason, you are the first one to make a github issue on it
… I need to figure out how to get loom and mixin to work properly at some point

also, little tiles is something _entirely_ different
Oh wait you edited it

**Comment by Spirit72** [comment]
yeah i realized i mixed up the names, they're quite similar haha whoops

i hope the mod works soon! this is the first time I've seen a mod that might help me make things as the inchling origin for the minecraft origins mod :DDD

**Comment by GiantLuigi4** [comment]
https://cdn.discordapp.com/attachments/892136637604171806/1098400084720365609/smallerunits-1.18.2-fabric-2.1.1.jar
should load, but I'm currently working on backporting the changes from forge
when I tested it, world loading got into a deadlock, so I'm not really confident in putting it on CF yet

**Comment by GiantLuigi4** [comment]
Update on that: I have it ported, but currently block entities are invisible, and even if that wasn’t the case, I had to get off for the day
Will hopefully have a CF build… soon(?)
Depends on fabric loom can function and if mappings don’t continue to get in the way of stuff

**Comment by GiantLuigi4** [comment]
https://curseforge.com/minecraft/mc-mods/smaller-units/files/4505810
File's still pending, but a fix has been uploaded @Spirit72
